### PR TITLE
docs(ISD-525): Documentation refactoring

### PR DESCRIPTION
**Pull request by amandahla**
* Rename tutorials to tutorial
* Rename how-to-guides to how-to
* Rename submenus to action ("how-to/configure-ldap.md")
* Rename how-to titles
* Capitalization - First letter of each heading only (European style)

### CODE REVIEW

File: docs/how-to/configure-external-hostname.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Configure the external hostname
+# How to configure external hostname
```
Comment (gtrkiller) [inline docs/how-to/configure-external-hostname.md:1]: Would it be worth to add an article (configure an/the external...) to this title?

File: docs/how-to/configure-proxy.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Proxy configuration
+# How to configure proxy
```
Comment (gtrkiller) [inline docs/how-to/configure-proxy.md:1]: same as above

File: docs/how-to/customize-theme.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Theme customization
+# How to customize theme
```
Comment (gtrkiller) [inline docs/how-to/customize-theme.md:1]: theme -> themes maybe?

File: docs/how-to/configure-external-hostname.md (hunk 1 above)
Comment (amandahla) [reply docs/how-to/configure-external-hostname.md:1]: Sure :)

**Review by jdkandersson — APPROVED** [review]

**Review by nrobinaubertin — APPROVED** [review]

File: docs/tutorial/getting-started.md [hunk 4]
```diff
@@ -80,7 +80,7 @@ indico                 3.2                           active      1  indico
 
 The deployment finishes when the status shows "Active".
 
-### Relate to Ingress by using NGINX Ingress Integrator
+### Relate to Ingress by using NGINX Ingress Integrator charm
```
Comment (gregory-schiano) [inline docs/tutorial/getting-started.md:83]: Do we keep "Ingress" and "Integrator" capitalized ?

**Pull request merged**